### PR TITLE
Update ES dvb-t frequencies

### PR DESCRIPTION
**Pull request by trujulu**

### CODE REVIEW

File: dvb-t/es-Almeria [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	FREQUENCY = 578000000
 	BANDWIDTH_HZ = 8000000
 	CODE_RATE_HP = 2/3
-	CODE_RATE_LP = NONE
+	CODE_RATE_LP = 2/3
```
Comment (perexg) [inline dvb-t/es-Almeria]: I don't think that this is right. Hierarchy is not set thus LP value is unused and should be really set to NONE. I fixed the autodiscovery in tvh to use NONE value when hierarchy is unused. v4.1-2112-gaf333d9

**Comment by perexg** [comment]
The commit with file removal was merged.

**Comment by trujulu** [comment]
But when you have it set to NONE, mux changes itself to 2/3 in Low LP. Is it normal?

2016-06-07 14:31:44.329 mpegts: mux 482MHz in TDT old params from DVB-T freq 482000000 bw 8MHz cons QAM/64 hier NONE **code_rate 2/3:NONE** guard 1/4 trans 8k plp_id -1 (00000020)
2016-06-07 14:31:44.329 mpegts: mux 482MHz in TDT new params to   DVB-T freq 482000000 bw 8MHz cons QAM/64 hier NONE **code_rate 2/3:2/3** guard 1/4 trans 8k plp_id -1 (00000020)

**Comment by perexg** [comment]
Upgrade your tvh. As I wrote, it's fixed since v4.1-2112-gaf333d9 .

**Pull request closed without merging**